OpenSnitch-Windows/build.ps1

```
@@ -0,0 +1,43 @@
+
+Write-Host "--- OpenSnitch-Windows Build Process ---"
+
+if ($BuildDaemon) {
+    Write-Host "Building Go Daemon..."
+    cd $WorkingDir\daemon
+    go mod tidy
+    # Build for Windows x64
+    $env:GOOS = "windows"
+    $env:GOARCH = "amd64"
+    go build -o ..\bin\opensnitchd.exe main.go process.go wfp.go decision.go grpc.go
+    if ($?) { Write-Host "Daemon build successful." -ForegroundColor Green }
```

PSScriptAnalyzer (1.24.0)

[warning] File 'build.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)